samples/server/go-server/api/server/config/config.go

```diff
@@ -67,6 +67,11 @@ func LoadConfig(fileName string) (*config, error) {
 		return nil, err
 	}
 	config := configFile.Grafeas
+
+	if config.StorageType == "memstore" {
```

Comment:
Great addition! I'd prefer the if clause to test for postgres (instead of memstore) and wrap the key creation code below. While it doesn't matter much today this will be better when other changes are made (like adding another storage type).